src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -936,19 +936,13 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 		}
 	}
 
-	generateCustomUploadImage(image, formdata) {
+	generateCustomUploadImage(image) {
 		// the imageUploaded-event of the angular image component is also called after initialising the component because the image is set in initFormFromExisting
 		if (typeof this.currentImage == 'undefined' || this.initedCurrentImage) {
 			this.initedCurrentImage = true;
 			this.currentImage = image.slice();
-			// Hide hexagon-frame if checkbox is checked
-			if (this.hideHexFrame) {
```

Comment:
Is the `hideHexFrame` property actually still used? Otherwise it would be nice to also remove it, to not cause confusion..